packages/amplify_datastore/lib/types/DataStoreHubEvents/HubEventElement.dart

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
should we start saying 2021 at least for the new files? I don't know what the impact really is